clients/hadoopfs/src/main/java/io/lakefs/Constants.java

```diff
@@ -2,9 +2,8 @@
 
 import org.apache.commons.io.FileUtils;
 
-class Constants {
+public class Constants {
```

Comment:
Why do we need a public class?

Reply:
think it me while writing a test code and want to use these constants - removed